examples/Calibration/Calibration.ino

```diff
@@ -2,7 +2,7 @@
 
 // enter your own sensor properties here, including calibration points
 FlowSensorProperties MySensor = {60.0f, 5.5f, {1, 1, 1, 1, 1, 1, 1, 1, 1, 1}};
-FlowMeter Meter = FlowMeter(2, MySensor);
+FlowMeter *Meter;
```

Comment:
Good idea to make this more dynamic, also thanks for alle the work you put into updating the examples accordingly! :)

I'd still have to check support in the Arduino IDE though.
I had tried doing it this way initially – it didn't work back then...

Reply:
For sure, and interesting it didn't work that way. Arduino IDE shouldn't have an issue with pointers. Any idea what error you ran into?

Reply:
C++ is fully supported within libraries, but the `.ino` files were handled differently by the preprocessor that was provided by the IDE. 

I'll check wether this is still the case.